parseedn.el

```diff
@@ -103,6 +186,8 @@ handlers as an optional argument to the reader functions.")
         (parseedn-print-kvs next)))))
 
 (defun parseedn-print (datum)
+  "Insert DATUM as EDN into the current buffer.
+DATUM can be any Emacs Lisp value."
```

Comment:
Is this true? not entirely sure. If it isn't, maybe we should add a `t` clause at the end that catches possible errors.